x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/indicator_match_rule.cy.ts

```diff
@@ -525,14 +525,9 @@ describe('indicator match', { tags: ['@ess', '@serverless', '@skipInServerlessMK
 
         cy.get(INDICATOR_MATCH_ROW_RENDER).should(
           'have.text',
-          `threat.enrichments.matched.field${
-            getNewThreatIndicatorRule().threat_mapping[0].entries[0].field
-          }${accessibilityText}matched${
-            getNewThreatIndicatorRule().threat_mapping[0].entries[0].field
-          }${
+          `${getNewThreatIndicatorRule().threat_mapping[0].entries[0].field}matched${
             indicatorRuleMatchingDoc.atomic
-          }${accessibilityText}threat.enrichments.matched.typeindicator_match_rule${accessibilityText}provided` +
```

Comment:
Hey, why we remove this accessibilityText?

Reply:
The accessibility text was related to draggables in Event Renderers and they are no longer draggables. I have created a follow-up issue to add a new text for all Event Renderers and not just this : https://github.com/elastic/kibana/issues/196505